src/lib/mina_base/zkapp_command.ml

```diff
@@ -1440,87 +1560,99 @@ let to_json x = Fields_derivers_zkapps.(to_json (deriver @@ Derivers.o ())) x
 
 let of_json x = Fields_derivers_zkapps.(of_json (deriver @@ Derivers.o ())) x
 
-let other_parties_of_json x =
+let account_updates_of_json x =
   Fields_derivers_zkapps.(
-    of_json ((list @@ Party.Graphql_repr.deriver @@ o ()) @@ derivers ()))
+    of_json
+      ((list @@ Account_update.Graphql_repr.deriver @@ o ()) @@ derivers ()))
     x
 
-let parties_to_json x =
+let zkapp_command_to_json x =
   Fields_derivers_zkapps.(to_json (deriver @@ derivers ())) x
 
 let arg_query_string x =
   Fields_derivers_zkapps.Test.Loop.json_to_string_gql @@ to_json x
 
 let dummy =
-  let party : Party.t =
-    { body = Party.Body.dummy; authorization = Control.dummy_of_tag Signature }
+  let account_update : Account_update.t =
+    { body = Account_update.Body.dummy
+    ; authorization = Control.dummy_of_tag Signature
+    }
   in
-  let fee_payer : Party.Fee_payer.t =
-    { body = Party.Body.Fee_payer.dummy; authorization = Signature.dummy }
+  let fee_payer : Account_update.Fee_payer.t =
+    { body = Account_update.Body.Fee_payer.dummy
+    ; authorization = Signature.dummy
+    }
   in
   { fee_payer
-  ; other_parties = Call_forest.cons party []
+  ; account_updates = Call_forest.cons account_update []
   ; memo = Signed_command_memo.empty
   }
 
-(* Parties transactions are filtered using this predicate
+(* Zkapp_command transactions are filtered using this predicate
    - when adding to the transaction pool
    - in incoming blocks
 *)
 let valid_size ~(genesis_constants : Genesis_constants.t) t : unit Or_error.t =
   let events_elements events =
     List.fold events ~init:0 ~f:(fun acc event -> acc + Array.length event)
   in
-  let ( num_proof_parties
-      , num_parties
+  let ( num_proof_zkapp_command
+      , num_zkapp_command
```

Comment:
nit: `num_updates`, `num_proof_updates` (if we accept `updates` as a short-hand for `account_updates` sometimes)